### ISSUE TITLE
Xcode Beta 14.5 - Stored properties cannot be marked potentially unavailable with '@available'

### ISSUE THREAD

**Issue by mmilakovic** [braintree/braintree_ios#854]
**Integration Details (please complete the following information):**
 - Braintree SDK Version: 5.11.0
 - Environment: Sandbox
 - iOS Version: iOS 15.5
 - Xcode Version: **Xcode Beta 14.5**
 - Device: all
 - Integration type & version:  Swift Package Manager 5.2.0
 - Development Processor Type: Intel

**Describe the bug**
We're unable to build the project on Xcode 14.5 beta with Braintree SPM (v 5.11.0) included. The error is:
`Stored properties cannot be marked potentially unavailable with '@available'`
in class `WebAuthenticationSession`.
```
    @available(iOS 13.0, *)
    private(set) lazy var presentationContextProvider: ASWebAuthenticationPresentationContextProviding? = nil
```

**To Reproduce**
3. See error

**Expected behavior**
The project with Braintree SPM should be buildable on all Xcode versions

**Screenshots**

<img width="1199" alt="Screenshot 2022-08-22 at 14 39 28" src="https://user-images.githubusercontent.com/6659472/185923170-4bc38fa9-02ce-4d2d-a695-6f526ba35dd3.png">

**Comment by jaxdesmarais** [comment]
Hello @mmilakovic -

Thanks for reporting this issue. Based on the [Xcode 14.5 Beta release notes](https://developer.apple.com/documentation/xcode-release-notes/xcode-14-release-notes), it mentions the following:

> Stored properties in Swift can’t have type information that is potentially unavailable at runtime. However, prior to Swift 5.7 the compiler incorrectly accepted @available attributes on stored properties when the property had either the lazy modifier or an attached property wrapper. This could lead to crashes for apps running on older operating systems. The Swift compiler now consistently rejects @available on all stored properties. (82713248) (FB9594187)

We will work to get this updated before Xcode 14 is officially released.

**Comment by jaxdesmarais** [comment]
Hey @mmilakovic -

The updates for iOS 16 + Xcode 14 have been released in [version 5.12.0 of the iOS SDK](https://github.com/braintree/braintree_ios/releases/tag/5.12.0). Please don't hesitate to reach back out if you are running into any other issues.

**Comment by mmilakovic** [comment]
@jaxdesmarais  Thanks for fixing it and for notifying us.
